src/openms/include/OpenMS/QC/MzCalibration.h

```diff
@@ -0,0 +1,65 @@
+// --------------------------------------------------------------------------
```

Comment:
duplicate file?
MzCalibration.h vs MZcalibration.h  ... 
MzCalibration.h should stay (correct name by coding convention)